app/src/main/java/com/example/myweatherapp/mvp/model/MyWeatherAppModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.myweatherapp.mvp.model
+
+import com.example.myweatherapp.mvp.contract.MyWeatherAppContract
+
+class MyWeatherAppModel: MyWeatherAppContract.MyWeatherAppModel {
+
+    override fun getData() {
+        TODO("Not working yet")
```

Comment:
when you run the app, this not crash here?

Reply:
yes, this crash.

Reply:
Done thx guys